### PR TITLE
Voice Over MacOS repeating bulleted list items

### LINKED ISSUE #6557: P2 A11y - Voice Over MacOS repeating bulleted list items on STY pages
**Describe the bug**
Following on from the full page STY a11y swarm, when navigating through a bulleted list on a STY page, some of the items are available twice when using Voiceover for MacOS on Safari e.g. some items are 2 key presses reading out the same content twice though with silighly different semantics, and the items that are 2 key presses are different depeding if you move through the content from the top or bottom of the component.

**To Reproduce**
Steps to reproduce the behaviour:

1. Activate VoiceOver for Mac
2. Visit https://www.test.bbc.com/gahuza/23302543
3. Navigate to the bulleted list with the screen reader navigation (Ctrl + alt/option + left/right arrow key)

**Expected behaviour**
Bulleted list items should only be read out once when navigating forwards and backwards.

**Testing notes**

- [x] This bug fix is expected to need manual testing.

**Additional context**
Check the behaviour with all other supported screen readers is ok.

### PR DESCRIPTION
**Pull request by Simon2828**
Resolves https://github.com/bbc/simorgh/issues/6557

**Overall change:** 
Remove `display:inline-block` from pseudo-element to prevent VoiceOver MacOS repeating bulleted list items.

**Code changes:**

- Remove `display: inline-block`
- Add padding round bullet point.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [X] This PR requires manual testing

**To manual test:**
- Use VoiceOver in Safari in Psammead Storybook on `BulletedList` component
- Use the `voiceover-list` branch in Simorgh. Link/copy the `psammead-bulleted-list` component into node_modules and test the bullet points on http://localhost:7080/gahuza/23302543

### CODE REVIEW

File: packages/components/psammead-bulleted-list/src/index.jsx [hunk 1]
```diff
@@ -8,6 +8,8 @@ import {
 } from '@bbc/gel-foundations/spacings';
 import { scriptPropType } from '@bbc/gel-foundations/prop-types';
 
+const BULLET_PADDING = '1.75rem';
```
Comment (12) [inline packages/components/psammead-bulleted-list/src/index.jsx:11]: Just checking this value has been confirmed by UX?

**Review by 12 — APPROVED** [review]
LGTM so long as my comment it is satisfied 🙂

**Comment by greenc05** [comment]
Please test with all supported screen readers @Simon2828 - Harvey should be able to help with what you don't have access to

**Review by FK78 — CHANGES_REQUESTED** [review]
The CHANGELOG.md, package-lock.json and package.json need to be bumped.

**Comment by Simon2828** [comment]
> The CHANGELOG.md, package-lock.json and package.json need to be bumped.

Should Talos handle this? If not, what do I need to do please? 
Thanks @rhenshaw56 for talking me through this 👍

File: packages/components/psammead-bulleted-list/src/index.jsx (hunk 1 above)
Comment (Simon2828) [reply packages/components/psammead-bulleted-list/src/index.jsx:11]: UX have confirmed it's ok 👍

**Review by HarveyPeachey — APPROVED** [review]
LGTM, tested on a11y laptop screen readers and they are behaving fine

**Comment by Simon2828** [comment]
Tested on talkback on Android and is working

**Review by FK78 — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
Looks good to me..

**Pull request merged**